LayoutTests/http/tests/inspector/target/provisional-load-cancels-previous-load-expected.txt

```diff
@@ -7,10 +7,10 @@ Current target is 0.
 PASS: Should receive TargetAdded event for target 1.
 PASS: Target 1 should be provisional.
 PASS: Target 1 should be paused on start.
-PASS: Should receive TargetRemoved event for target 1
 PASS: Should receive TargetAdded event for target 2.
 PASS: Target 2 should be provisional.
 PASS: Target 2 should be paused on start.
+PASS: Should receive TargetRemoved event for target 1
```

Comment:
Did behavior change here on purpose?

Reply:
Presumably I extended the lifetime of an object and it delayed some IPC. The test is still passing so it didn't seem particularly harmful.